gitlab/integration.sh

```diff
@@ -175,61 +192,68 @@ export CURLOPTS="--fail -sq -m 30 -b $COOKIEJAR"
-
-# Send a general clarification to later test if we see the event.
-curl $CURLOPTS -F "sendto=" -F "problem=1-" -F "bodytext=Testing" -F "submit=Send" \
-     "http://localhost/domjudge/jury/clarifications/send" -o /dev/null
-
-# Don't spam the log.
-set +x
-
-while /bin/true; do
-    sleep 30s
-    curl $CURLOPTS "http://localhost/domjudge/jury/judging-verifier?verify_multiple=1" -o /dev/null
-
-    # Check if we are done, i.e. everything is judged or something got disabled by internal error...
-    if tail /tmp/judgedaemon.log | grep -q "No submissions in queue"; then
-        break
-    fi
-    # ... or something has crashed.
-    if ! pgrep -f judgedaemon; then
-        break
-    fi
-done
-
-NUMNOTVERIFIED=$(curl $CURLOPTS "http://localhost/domjudge/jury/judging-verifier" | grep "submissions checked"     | sed -r 's/^.* ([0-9]+) submissions checked.*$/\1/')
-NUMVERIFIED=$(   curl $CURLOPTS "http://localhost/domjudge/jury/judging-verifier" | grep "submissions not checked" | sed -r 's/^.* ([0-9]+) submissions not checked.*$/\1/')
-NUMNOMAGIC=$(    curl $CURLOPTS "http://localhost/domjudge/jury/judging-verifier" | grep "without magic string"    | sed -r 's/^.* ([0-9]+) without magic string.*$/\1/')
-section_end judging
-
-# We expect
-# - two submissions with ambiguous outcome,
-# - one submissions submitted through the submit client, and thus the magic string ignored,
-# - and all submissions to be judged.
-if [ $NUMNOTVERIFIED -ne 2 ] || [ $NUMNOMAGIC -ne 1 ] || [ $NUMSUBS -gt $((NUMVERIFIED+NUMNOTVERIFIED)) ]; then
-    section_start error "Short error description"
-    # We error out below anyway, so no need to fail earlier than that.
-    set +e
-    echo "verified subs: $NUMVERIFIED, unverified subs: $NUMNOTVERIFIED, total subs: $NUMSUBS"
-    echo "(expected 2 submissions to be unverified, but all to be processed)"
-    echo "Of these $NUMNOMAGIC do not have the EXPECTED_RESULTS string (should be 1)."
-    curl $CURLOPTS "http://localhost/domjudge/jury/judging-verifier?verify_multiple=1" | w3m -dump -T text/html
-    section_end error
-
-    section_start logfiles "All the more or less useful logfiles"
-    for i in /opt/domjudge/judgehost/judgings/*/*/*/*/*/compile.out; do
-        echo $i;
-        head -n 100 $i;
-        dir=$(dirname $i)
-        if [ -r $dir/testcase001/system.out ]; then
-            head $dir/testcase001/system.out
-            head $dir/testcase001/runguard.err
-            head $dir/testcase001/program.err
-            head $dir/testcase001/program.meta
+section_end curlcookie
+
+if [ $cgroupv1 -ne 0 ]; then
+    section_start judging "Waiting until all submissions are judged"
+    # wait for and check results
+    NUMSUBS=$(curl --fail http://admin:$ADMINPASS@localhost/domjudge/api/contests/1/submissions | python3 -mjson.tool | grep -c '"id":')
+    
+    # Send a general clarification to later test if we see the event.
+    curl $CURLOPTS -F "sendto=" -F "problem=1-" -F "bodytext=Testing" -F "submit=Send" \
```

Comment:
```suggestion
    curl $CURLOPTS -F "sendto=" -F "problem=1-" -F "bodytext=Testing" -F "submit=Send" \
```

Like a said, we can test this with the evenfeed still if its received.

Reply:
GitHub (and I) don't see any changes suggested here. But I assume I addressed the suggestion anyway by moving it outside of the if.